rocAL/rocAL_pybind/examples/tf_petsTrainingExample/train_withROCAL_withTFRecordReader.py

```diff
@@ -142,13 +142,14 @@ def main():
                                               mean=[0,0,0],
```

Comment:
Are these the right values for mean and std?

Reply:
The mean and std for the CMN was 0 and 255 so I added the same values for pipeline initialization